lib/osf-components/addon/components/resources-list/template.hbs

```diff
@@ -4,7 +4,13 @@
         data-test-add-resource-section
     >
         <span>{{t 'osf-components.resources-list.add_instructions'}}</span>
-        {{!-- help icon --}}
+        <OsfLink
+            aria-label={{t 'osf-components.resources-list.help_doc_link'}}
+            local-class='HelpLink'
+            @href='https://help.osf.io/article/410-registration-files'
+        >
```

Comment:
Probably could use a data-test attribute in case QA wants to selenium test this.